{"success": true, "data": [{"type": "correct", "user": {"id": 396, "name": "mathcuber"}, "challenge_id": 54, "date": "2024-04-26T17:41:18.946210+00:00", "id": 1588, "challenge": {"id": 54, "name": "sanitycheck", "value": 1, "category": "misc"}, "team": {"id": 227, "name": "errormath"}}], "meta": {"count": 1}}
